functions/lib/stripe/webhooks.py

```
@@ -300,6 +300,11 @@ def stripe_webhook_checkout_fulfilment(req: https_fn.Request) -> https_fn.Respon
       logger.info(f"Ignoring event. event={event}")
       return https_fn.Response(status=200)
 
+  SPORTSHUB_URL = "www.sportshub.net.au"
+  if not SPORTSHUB_URL in event["id"]["object"]["cancel_url"] and not SPORTSHUB_URL in event["id"]["object"]["success_url"]:
```

Ruff (0.12.2)

Test for membership should be `not in`

Convert to `not in`

(E713)